### PR TITLE
[DOC] shopfloor: delivery + zone_picking sequence diagrams

### PR DESCRIPTION
**Pull request by sebalix**
A basic sequence diagram for the `delivery` and `zone_picking` scenarios.
Doesn't include all possible state redirections in case of errors (bad ID sent, etc), it shows only the meaningful ones.

### CODE REVIEW

File: shopfloor/docs/zone_picking_diag_seq.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+set_line_destination -> select_line: **/set_destination**
+set_line_destination -> zero_check: **/set_destination**
+zero_check -> select_line: **/is_zero**
+select_line -> unload_single: **/prepare_unload** \n(different destinations in buffer lines, process one by one)
+select_line -> unload_set_destination: **/prepare_unload** \n(only one move line in the buffer)
+select_line -> unload_all: **/prepare_unload** \n(buffer lines have all the same destination location)
+select_line -> select_line: **/prepare_unload** \n(no remaining lines in the buffer)
+unload_all -[#red]> unload_all: **/set_destination_all** \n(error, scanned destination location invalid)
+unload_all -> unload_all: **/set_destination_all** \n(scanned destination not expected but valid, confirmation required)
+unload_all -> select_line: **/set_destination_all**
```
Comment (guewen) [inline shopfloor/docs/zone_picking_diag_seq.txt]: `unload_all` should have a return to `start` when all lines are done

**Review by guewen — COMMENTED** [review]
:clap: :hugs:

Maybe add a comment in the zone picking service docstring with a pseudo-link to the doc?

File: shopfloor/docs/zone_picking_diag_seq.txt (hunk 1 above)
Comment (sebalix) [reply shopfloor/docs/zone_picking_diag_seq.txt]: done, plus:
+ added the call to `/select_zone` when the `start` screen is opened
+ added separators between endpoints, could make things clearer

**Comment by sebalix** [comment]
`delivery` scenario added.

Still don't know why runbot is complaining, I didn't succeed to access its log.

**Comment by simahawk** [comment]
@sebalix is not red, is yellow because of 
`2020-11-10 12:46:40,002 171 WARNING openerp_test odoo.addons.base.models.decimal_precision: Deprecated call to decimal_precision.get_precision(<application>), use digits=<application> instead`

**Comment by simahawk** [comment]
@sebalix can you rebase and squash? Let's merge this if there's no blocker.

**Comment by sebalix** [comment]
@simahawk squashed & rebased

**Review by guewen — APPROVED** [review]

**Pull request merged**